src/app/components/socialSection/socialLinks.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from 'next/image';
+import { SocialLinkData } from './socialSection';
+import styles from './socialSection.module.css';
+
+interface SocialLinksProps {
+  links: SocialLinkData[];
+}
+
+const SocialLinks = ({ links }: SocialLinksProps) => {
+  return (
+    <ul className={styles.socialLinks}>
+      {links.map((link) => (
+        <li key={link.id}>
+          <a href={link.link} target='_blank'>
+            <Image src={link.imgSrc} alt={link.alt} width={100} height={100} />
```

Comment:
Curious for thoughts on using [React icons](https://react-icons.github.io/react-icons/) in place of the static images. This could give more control over the base and hover colors of the icons and give more consistency across all the social icons.

Reply:
I agree. Can you proceed to tackle this?